packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -26,6 +26,7 @@ abstract class AnimatedComponent extends StatefulComponent {
 
   Widget build(BuildContext context);
 
+  /// Subclasses are advised to not override this method.
```

Comment:
There's nothing particularly wrong with overriding this function.  It's just not very useful.  Maybe say that "Subclasses typically do not override this method."